src/proficiency.cpp

```diff
@@ -179,7 +179,12 @@ float proficiency::default_weakpoint_penalty() const
 
 time_duration proficiency::time_to_learn() const
 {
-    return _time_to_learn;
+    const double scaling = get_option<float>( "SKILL_TRAINING_SPEED" );
+    if( scaling != 1.0 && scaling != 0.0 ) {
```

Comment:
Is the 0.0 option also ignored in the other uses of the setting? Shouldn't 0 mean no learning?

Reply:
if we remove the "&& scaling != 0.0" would that work

Reply:
yes 0 should mean no learning it shouldnt ignored

Reply:
Fixed with the next commit:  when the option is 0.0 there is no progress on proficiency anymore and the UI is more consistent.